flake.nix

```diff
@@ -51,17 +36,8 @@
       url = "github:GNOME/gnome-shell/47.2";
     };
 
-    # The 'home-manager' input is used to generate the documentation.
-    home-manager = {
-      inputs.nixpkgs.follows = "nixpkgs";
-      url = "github:nix-community/home-manager";
-    };
-
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
 
-    # Interface flake systems.
-    systems.url = "github:nix-systems/default";
-
```

Comment:
Obviously, revert this removal along with https://github.com/danth/stylix/pull/1208/files#r2080431102